packages/fcl/src/current-user/index.js

```diff
@@ -116,6 +119,25 @@ async function getAccountProofData() {
   return accountProofData
 }
 
+const makeDiscoveryServices = async () => {
+  const extensionServices = window?.fcl_extensions || []
+  return [...extensionServices, ...serviceRegistry.getServices()]
```

Comment:
Should we shuffle these?

Reply:
reverse order?

Reply:
Maybe let's just leave as is since Discovery will handle it